.gitlab-ci.yml

```diff
@@ -59,17 +65,22 @@ package-production:
     only:
         - tags
 
-# Not executed, parent job definition for Tomcat deployments
+# Not executed, parent job definition for deployments
 .deploy:
     stage: deploy
     image: alpine:3.9.5
     environment:
         name: $ENVIRONMENT_NAME
     script:
-        - apk add --update curl
-        - curl -u $TOMCAT_USER:$TOMCAT_PASSWORD -T "$ARTIFACT_PATH" "http://$TOMCAT_HOST/manager/text/deploy?update=true&path=/eva/webservices/identifiers&version=$DATETIME" | grep "OK - Deployed application"
+        - apk add --update curl openssh-client
+        - curl -u $TOMCAT_USER:$TOMCAT_PASSWORD -T "$ACC_WS_ARTIFACT_PATH" "http://$TOMCAT_HOST/manager/text/deploy?update=true&path=/eva/webservices/identifiers&version=$DATETIME" | grep "OK - Deployed application"
+        # Start ssh-agent and set identity then copy the Jar file and create a symlink
+        - eval "$(ssh-agent)"; echo "$EVA_ETL_PRIVATE_KEY" > identity_id_rsa; chmod 600 identity_id_rsa; ssh-add identity_id_rsa
```

Comment:
Can this be simplified? also it can be split in different lines so its more readable, maybe something like:
```
        - eval $(ssh-agent)
        - echo "$EVA_ETL_PRIVATE_KEY" | ssh-add -
```

Reply:
That does improve readability